doc/SpecifyingApplications/TestCases/TestCases.md

```diff
@@ -1,14 +1,101 @@
-# Test Cases
-
-Individualization
-•	Find set of common test cases here 
-•	Load common test case collection into Postman
-•	Create DATAfile from LOADfile
-•	Delete unnecessary segments
-Defining Test Cases
-•	Create folder structures for individual services
-•	Add standard tests for individual services
-•	Create folders for individual tests of individual services
-•	Describe individual tests
-•	Implement individual tests
-•	Execute tests against mock server
+# Testcases
+
+## Folder structure
+
+The collection contains sub-folders like UserInput, functions(), ServiceLayer, OamLayer, Termination folders
+
+![Folder Structure](./pictures/folderStructure.PNG)
+
+### UserInput
+
+Here the data from the input load-file is extracted and stored for using it in proceeding testcases.
+
+*Reference:* [*Load File*](../LoadFile/LoadFile.md)
+
+### functions()
+
+In this folder, any common re-usable functions could be defined.
+
+#### Some keywords used for defining the requests *(for reference)*
+
+|**Prefix keyword**|**Purpose of test scripts**|
+| :---- | :---- |
+|Reference|Gathering some required information related to the proceeding test script|
+|Expected|Testing expected scenario specific testcases|
+|Dummy|POST/PUT some randomly generated dummy values through into request body|
+|Initial|POST/PUT service/API with Initial values for inversing any potential change in configuration|
+|Clearing|Operations performed for clearing any evident changes in the configuration|
+|sampling|Testing the actual testcase (just for experimental purpose)|
+|Check|Test script that test whether the configuration remains the same as initial data|
+
+### ServiceLayer and OamLayer
+
+This folder contains testcases for API enpoints exposed in serviceLayer. It further contains IndividualPart and BasicPart which refers to IndividualServices and BasicServices respectively in OAS specification. Under IndividualPart or BasicPart, testcases for all the services or AI endpoints are listed.
```

Comment:
"**AI** endpoints are listed." to "**API** endpoints are listed."

Reply:
updated as per comment